storage/src/adb/any/fixed/sync.rs

```diff
@@ -1016,8 +1006,7 @@ mod tests {
             update_sender
                 .send(Target {
                     root: initial_root,
-                    lower_bound: initial_upper_bound, // Greater than upper bound
-                    upper_bound: initial_lower_bound, // Less than lower bound
+                    range: initial_upper_bound..=initial_lower_bound, // Invalid: lower > upper
```

Comment:
update comment?

Reply:
I just removed this. There's a comment 4 lines above that explains this already